src/api/Cargo.toml

```diff
@@ -33,7 +33,6 @@ jwt-simple = "0.11.4"
 kbs-types = "0.4"
 lazy_static = "1.4.0"
 log.workspace = true
-mime.workspace = true
```

Comment:
I'm not sure this is related to this commit?

Reply:
Oh, let me fix this. It is not used.